tarantool_test.go

```diff
@@ -52,6 +52,34 @@ func (m *Member) decodeMsgpackImpl(d *Decoder) error {
 	return nil
 }
 
+func convertUint64(v interface{}) (result uint64, err error) {
```

Comment:
I tried to replace the use of `uint` with `string` where possible to avoid such conversions.

The call now exists only in tarantool_test.go.  I think it should be there:

1. We highlight the problem for developers.
2. We can refer to the problem in our tests.